keps/sig-node/20190717-seccomp-ga.md

```diff
@@ -350,7 +395,37 @@ However, to raise awareness of existing controllers using the seccomp annotation
 migrated, the same warning annotation will be added to the controller as for pods:
 
 ```
-warning.kubernetes.io/seccomp: "Seccomp set through annotations. Support will be dropped in v1.22"
+warning.kubernetes.io/seccomp: "Seccomp set through annotations. Support will be dropped in v1.23"
+```
+
+#### docker/default
+
+The runtime profile `docker/default` was deprecated as of Kubernetes 1.11. However, the API server 
+will still support this profile type when migrating existing seccomp annotations. 
+
+An additional warning annotation will be added when a pod is created using the `docker/default` profile:
+```
+warning.kubernetes.io/seccomp: "Seccomp profile docker/default is deprecated. Support will be dropped in v1.23"
+```
+
+#### Runtime Profiles 
+
+The API Server will continue to reject annotations with runtime profiles different than `runtime/default`, 
+to maintain the existing behavior. 
+
+Violations would lead to the error message:
+```
+Invalid value: "runtime/profile-name": must be a valid seccomp profile
+```
+
+#### Localhost Profile Validation
+
+The API Server will start to reject annotations with localhost profiles which do not comply with the 
+new validation requirements defined in [Pod API](#pod-api). 
```

Comment:
This is tightening validation, which is generally problematic. See https://github.com/kubernetes/kubernetes/issues/64841

I think this is easier since the new field will be immutable. I think we can apply the tightened validation on create, and ignore the validation on update.
/cc @liggitt

Reply:
Updated based on recommendations. Thank you for adding the context.